lib/ts/recipe/emailpassword/api/implementation.ts

```diff
@@ -663,6 +675,15 @@ export default function getAPIImplementation(): APIInterface {
             }
 
             if (isFakeEmail(email) && preAuthChecks.isFirstFactor) {
+                // we will intercept this request in favor to detect brute force attacks
+                if (anomalyDetection !== undefined) {
+                    await anomalyDetection.recipeInterfaceImpl.checkAnomalyWithRequest({
+                        request: options.req,
+                        status: "WRONG_CREDENTIALS_ERROR",
```

Comment:
why do we care about checking for anomaly detection if the input credentials are wrong?

Reply:
the idea here is to have the ip / headers of a possible brute force attack.

Reply:
https://github.com/sandrodesouza/supertokens-node/pull/1#discussion_r1565494864